packages/plugin-ext/src/main/browser/authentication-main.ts

```diff
@@ -273,7 +254,7 @@ interface AccountUsage {
 
 export class AuthenticationProviderImpl implements AuthenticationProvider {
     private accounts = new Map<string, string[]>(); // Map account name to session ids
-    private sessions = new Map<string, string>(); // Map account id to name
+    private sessions = new Map<string, string>();   // Map account id to name
```

Comment:
Maybe as a JSDoc so that it appears on hover for anyone working here in the future?